index.html

```diff
@@ -7,6 +7,11 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="View soccer goals as they happen around the world" />
     <link rel="apple-touch-icon" href="/logo192.png" />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
```

Comment:
Fix for the white scrollbar on my local is to make this version 5.3.1, so get it from here https://getbootstrap.com/

Then in index.css, change the import at the top to be 5.3.1 as well
`@import 'https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css';`